books/rtl/rel11/support/verify-guards.lisp

```diff
@@ -518,14 +518,15 @@
    :hints (("goal" :in-theory (enable formatp sigw prec)))))
 
 (defund indef (f)
-  (declare (xargs :guard (formatp f)))
+  (declare (xargs :guard (formatp f)
+                  :verify-guards nil))
```

Comment:
Apparently `:verify-guards nil` was copied from `definitions.lisp` . It is not necessary here.